administrator/language/de-DE/com_content.ini

```
@@ -73,6 +73,7 @@ COM_CONTENT_FIELD_FULLTEXT="Kompletter Text"
 COM_CONTENT_FIELD_FULL_LABEL="Komplettes Beitragsbild"
 COM_CONTENT_FIELD_IMAGE_ALT_LABEL="Alternativer Text"
 COM_CONTENT_FIELD_IMAGE_CAPTION_LABEL="Bildunterschrift"
+COM_CONTENT_FIELD_IMAGE_CLASS_LABEL="Bildklasse"
```

```suggestion
COM_CONTENT_FIELD_IMAGE_CLASS_LABEL="CSS-Klasse des Bildes"
```

analog: https://github.com/joomlagerman/joomla/blob/5e64af980f7e291b2d40775a7ce072888c367781/language/de-DE/lib_joomla.ini#L360